app/src/main/res/drawable/d20_svg.xml

```diff
@@ -1,13 +1,14 @@
-<vector android:alpha="0.52"
```

Comment:
d20 isn't a very clear name and that its an svg doesn't matter much either when using it